groups-api/Groups API Sandbox (empty).postman_environment.json

```diff
@@ -0,0 +1,33 @@
+{
+	"id": "df1590e3-fcd6-4e88-8a3a-176e65aea4a0",
+	"name": "Groups API Sandbox (empty)",
+	"values": [
+		{
+			"key": "host",
+			"value": "https://api.sandbox.transferwise.tech",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "token",
+			"value": "",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "profile-id",
```

Comment:
Let's call it `profile_id` to be consistent with the other Postman collection.

Reply:
Changed!